Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionCountrySelectViewModelTest.kt

```diff
@@ -0,0 +1,59 @@
+package de.rki.coronawarnapp.ui.submission.viewmodel
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import de.rki.coronawarnapp.ui.submission.SubmissionCountry
+import io.kotest.inspectors.forAll
+import io.kotest.inspectors.forAtLeastOne
+import io.kotest.inspectors.forAtMostOne
+import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.shouldBe
+import org.junit.Rule
+import org.junit.Test
+
+class SubmissionCountrySelectViewModelTest {
+    @get:Rule
+    val instantTaskExecRule = InstantTaskExecutorRule()
+
+    @Test
+    fun testFetchCountries() {
+        val viewModel = SubmissionCountrySelectViewModel()
+
+        viewModel.fetchCountries()
+        // TODO: implement proper test one backend is merged
+        viewModel.countries.value?.shouldHaveSize(2)
```

Comment:
Try to avoid nullsafe `?` in tests, if the value is `null` which would be unexpected, the test statements don't fail.

Force it to be non null via `!!` so we get an exception that fails the test if this is null.